app/views/data_exports/_new_export_dialog.html.erb

```diff
@@ -0,0 +1,57 @@
+<%= viral_dialog(open: open, size: :large) do |dialog| %>
+  <%= dialog.with_header(title: t(".title")) %>
+  <%= dialog.with_section do %>
+    <%= form_for(:data_export, url: data_exports_path,
+        data: {
+          turbo: false
+        }, method: :post) do |form| %>
+    <div
+      data-controller="sessionstorage-amend-form"
+      data-sessionstorage-amend-form-target="field"
+      data-sessionstorage-amend-form-field-name-value="data_export[export_parameters[ids]][]"
+    />
+    <div class="mb-4 font-normal text-gray-500 dark:text-gray-400">
+      <div class="form-field mb-4">
+        <%= form.label :name, t(".name_label") %>
+        <%= form.text_field :name, class: "form-control",
+        data: {
+          'data-exports--create-target': 'nameInput'
+        } %>
+      </div>
+      <div class="flex mb-4 items-center h-5">
+        <%= form.label :email_notification, t(".email_label"),
```

Comment:
**nitpick:** Try moving the checkbox before the description, it is more natural to see a checkbox on the left side

Reply:
Moved to left side 8cfbfce670dc7830e1675947b000d13be9846edb